File: Java/1045.java

import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);

        double A, B, C, aux;

        A = input.nextDouble();
        B = input.nextDouble();
        C = input.nextDouble();

        if (B > A && B > C) {
            aux = B;
            B = A;
            A = aux;
        }
        if (C > A && C > B) {
            aux = C;
            C = A;
            A = aux;
        }

        if (A >= (B + C)) {
            System.out.println("NAO FORMA TRIANGULO");
            return;
        }

        if (A * A == ((B * B) + (C * C))) {
            System.out.println("TRIANGULO RETANGULO");
        }

        if (A * A > ((B * B) + (C * C))) {
            System.out.println("TRIANGULO OBTUSANGULO");
        }

        if (A * A < ((B * B) + (C * C))) {
            System.out.println("TRIANGULO ACUTANGULO");
        }

        if (A == B && B == C) {
            System.out.println("TRIANGULO EQUILATERO");
            return;
        }
        if (A == B || B == C || C == A) {
            System.out.println("TRIANGULO ISOSCELES");
        }

    }
}

File: Python/1045.py
import sys
A , B , C = list (map(float,input().split()))

if B > A and B > C:
    aux = B
    B = A
    A = aux

if C > A and C > B:
    aux = C
    C = A
    A = aux

if A >= B + C:
    print ("NAO FORMA TRIANGULO")
    sys.exit()

if A * A == (B * B + C * C):
    print ("TRIANGULO RETANGULO")

if A * A > ((B * B) + (C * C)):
    print ("TRIANGULO OBTUSANGULO")
    

if A * A < ((B * B) + (C * C)):
    print ("TRIANGULO ACUTANGULO")
    

if A == B and B == C:
    print ("TRIANGULO EQUILATERO")
    sys.exit()

if A == B or B == C or C == A:
    print ("TRIANGULO ISOSCELES")